opal/corelib/range.rb

```diff
@@ -85,16 +93,26 @@ def exclude_end?
     @exclude
   end
 
-  alias :first :begin
+  def first(n=(n_specified=true; nil))
```

Comment:
I'll just leave it up to you to change it to whichever form is most syntactically appropriate.

I know how `null/undefined` works in JS, and how `nil` and optional arguments work in Ruby, but I have no idea how they both interact in Opal.